magenta-lib/src/main/scala/magenta/Project.scala

```diff
@@ -130,14 +131,18 @@ object HostList {
   implicit def hostListAsListOfHosts(hostList: HostList) = hostList.hosts
 }
 
+case class RuntimeResources(reporter: DeployReporter, lookup: Lookup, parameters: DeployParameters, stack: Stack) {
```

Comment:
Yep - that did occur to me there were two things. You'd advocate for two case classes? `Resources` and `???`.